### DISCUSSION TITLE
upgrading from argo-workflows v2.4.2

### DISCUSSION THREAD

**Discussion by tanken** [argoproj/argo-workflows#10739] [Q&A]
Hi, i am using a helm chart argo-workflows-2.4.2 (app v3.3.9) and am planning to perform an update, was wondering if it's ok to update to the latest version or an incremental one. Im also using an external postgres database 14.5 in rds, was wondering if this has to be upgraded prior to the argo update as well.

**Comment by terrytangyuan** [comment]
I don’t have advice for helm chart upgrade but check out the upgrading notes here related to Argo controller/server: https://github.com/argoproj/argo-workflows/blob/master/docs/upgrading.md
